test/functional/flutter/commands_test.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from appium.webdriver.common.flutterby import FlutterBy
+from appium.webdriver.flutter_finder import FlutterFinder
+from test.functional.flutter.helper.test_helper import BaseTestCase
+
+class TestFlutterCommands(BaseTestCase):
+    
+    def test_wait_command(self):
+        self.__open_screen('Lazy Loading')
+    
+        message_field_finder = FlutterFinder.by_flutter_key('message_field')
+        toggle_button_finder = FlutterFinder.by_flutter_key('toggle_button')
+        
+        message_field = self.driver.find_element(*message_field_finder.as_strings())
+        toggle_button = self.driver.find_element(*toggle_button_finder.as_strings())
+        assert message_field.is_displayed() == True
+        assert message_field.text == 'Hello world'
+
+        toggle_button.click()
+        self.flutter_command.wait_for_invisible(message_field_finder)
+        assert len(self.driver.find_elements(*message_field_finder.as_strings())) == 0
```

Comment:
it probably makes sense to create a wrapper over find_elements, so we may directly pass Finder instance to it to avoid unnecessary duplication

Reply:
For now I think its not required. I have just wrote it as an example  **if** we want to use finder instance.